generators/app/templates/_.npmignore

```diff
@@ -0,0 +1 @@
+!dist/
```

Comment:
If I understand npmignore correctly, it means that only `dist/` will be included in the package for the module. I would like to include everything -- docs, sources, libs, tests -- and not just `dist/`.